CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * [#3570](https://github.com/bbatsov/rubocop/issues/3570): Add new `MultilineIfModifier` cop to avoid usage of if/unless-modifiers on multiline statements. ([@tessi][])
 * [#3631](https://github.com/bbatsov/rubocop/issues/3631): Add new `Style/SpaceInLambdaLiteral` cop to check for spaces in lambda literals. ([@swcraig][])
 * Add new `Lint/EmptyExpression` cop. ([@drenmi][])
+* [#3600](https://github.com/bbatsov/rubocop/issues/3600): Add new `Style/OrderedGems` cop. ([@tdeo][])
```

Comment:
This should be moved to the proper section of the changelog (we did a gem release recently). Also style should be replaced with Bundler.